### PR TITLE
TOMEE-4332 Load SpringWebUtils in the context class loader in TomEE w…

### PR DESCRIPTION
**Pull request by jgallimore**
…hen the cxf-rs service is initialized

### CODE REVIEW

**Comment by jgallimore** [comment]
Further detail on the CXF list is here: https://lists.apache.org/thread/coxrst45zm8gqmf1hhx5g7bqzbsldkhh

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/AlternativeGreeter.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.apache.openejb.arquillian.tests.jaxrs.spring;
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/AlternativeGreeter.java:1]: License header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/DemoApplication.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.apache.openejb.arquillian.tests.jaxrs.spring;
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/DemoApplication.java:1]: License header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/GreetingController.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package org.apache.openejb.arquillian.tests.jaxrs.spring;
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/GreetingController.java:1]: License header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/ServletInitializer.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package org.apache.openejb.arquillian.tests.jaxrs.spring;
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/ServletInitializer.java:1]: License header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/resources/spring-webmvc-pom.xml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/resources/spring-webmvc-pom.xml:1]: missing header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/resources/spring-webmvc-pom.xml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tomee</groupId>
+            <artifactId>jakartaee-api</artifactId>
+            <version>9.1.1</version>
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/resources/spring-webmvc-pom.xml:40]: For `main` this should be 10.0-M2

File: arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/SpringWebappTest.java [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+    @Test
+    public void validate() throws Exception {
+        final String launchProfile = System.getProperty("arquillian.launch");
+        if ("tomee-embedded".equals(launchProfile)) {
+            System.out.println("Skipping this test in TomEE embedded");
+            return;
+        }
+
+        final InputStream is = new URL(url.toExternalForm() + "hello").openStream();
```
Comment (rzo1) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/java/org/apache/openejb/arquillian/tests/jaxrs/spring/SpringWebappTest.java:107]: Wouldn't an IO.slurp(...) be sufficient here?

**Review by rzo1 — APPROVED** [review]
Fixed the license and the ee9 api ref. LGTM. Thanks @jgallimore

**Pull request merged**